README.md

```diff
@@ -10,9 +10,9 @@ Writen by Vitor Balocco ([@vitorbal](https://github.com/vitorbal)) and Adam Terl
 ## Usage
 
 ```javascript
-import { createBBStore, combineBBEntities } from 'simple-backbone-redux';
+import { bbCreateStore, bbCombineEntities } from 'simple-backbone-redux';
 
-const bbStore = createBBStore(combineBBEntities({
+const bbStore = bbCreateStore(bbCombineEntities({
```

Comment:
Yes.  This README update was just a find-replace for the old method changing names.  Really it's still a big TODO to update the main README with all the appropriate usage.